ote/src/cljs/ote/app/controller/taxi_prices.cljs

```diff
@@ -0,0 +1,63 @@
+(ns ote.app.controller.taxi-prices
+  (:require [tuck.core :as tuck :refer-macros [define-event]]
+            [tuck.effect :as tuck-effect]
+            [ote.app.state :as state]
+            [ote.localization :refer [tr tr-key]]
+            [ote.communication :as comm]
+            [ote.app.controller.common :refer [->ServerError]]))
+
+(defrecord LoadTaxiPrices [])
+(defrecord LoadTaxiPricesResponse [response])
+(defrecord ApproveByIds [pricing-ids])
+(defrecord ApproveByIdsResponse [response])
+
+(def every-5min (* 1000 60 1))
```

Comment:
I don't understand this. Why would `60000` be every 5 minutes? Do you mean (assuming time is in milliseconds) this:
`(def every-5min (* 1000 60 5))` ?

Reply:
Not my bug :-) Seems the place I copy-pasted this from has the same issue. Will fix.